### PR TITLE
Serving with seldon

### PR DESCRIPTION
**Pull request by jsuchome**
Adding new serving templates. Seldon can use different inference servers, so for now I'm adding 2 templates.
Alternative would be having one seldon template with more variables.

### CODE REVIEW

**Comment by jsuchome** [comment]
https://github.com/fuseml/fuseml/pull/29 is needed to fix (some) CI failures

File: paas/client.go [hunk 1]
```diff
@@ -172,6 +183,9 @@ func (c *FusemlClient) Apps() error {
 			}
 			routes = strings.Join(ingRoutes, ", ")
 		}
+		if routes == "" {
```
Comment (flaviodsr) [inline paas/client.go:186]: Is there a case where `routes` is empty?

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: embedded-files/serving/deployment.yaml.tmpl [hunk 2]
```diff
@@ -34,6 +34,8 @@ spec:
           args:
             - -c
             - |
+              export LC_ALL=C.UTF-8
+              export LANG=C.UTF-8
```
[the remark this answers is not in the document]
Comment (jsuchome) [reply embedded-files/serving/deployment.yaml.tmpl:38]: OK, will you adapt it?

File: paas/client.go (hunk 1 above)
Comment (jsuchome) [reply paas/client.go:186]: Ah ... this seems like some relic from testing...

File: embedded-files/serving/deployment.yaml.tmpl (hunk 2 above)
[the remark this answers is not in the document]
Comment (jsuchome) [reply embedded-files/serving/deployment.yaml.tmpl:38]: Or do you want me to adapt the Dockerfile in `paas/client.go` ?

[the remark this answers is not in the document]
Comment (jsuchome) [reply embedded-files/serving/deployment.yaml.tmpl]: Done in https://github.com/fuseml/fuseml/pull/36

**Review by flaviodsr — APPROVED** [review]
LGTM

**Pull request merged**